src/components/widgets-settings/attach-kpi/attach-kpi.component.coffee

```diff
@@ -0,0 +1,95 @@
+module = angular.module('impac.components.widgets-settings.attach-kpis', [])
```

Comment:
This component could be pretty flexible - are you sure you want it to just be for threshold? But yeah was working quickly, I have left the old component in there for reference too - that needs to be removed once update / delete are working, at that point let's talk about renaming it.